libs/server/Resp/AdminCommands.cs

```diff
@@ -488,6 +557,79 @@ private bool ProcessAdminCommands<TGarnetApi>(RespCommand command, ReadOnlySpan<
             return true;
         }
 
+
+        void ProcessHelloCommand(int? protocolVersion, ReadOnlySpan<byte> username, ReadOnlySpan<byte> password, string clientName)
+        {
+            if (protocolVersion != null)
+            {
+                if (protocolVersion.Value is < 2 or > 3)
```

Comment:
Should we allow a protocol switch to RESP3?

Reply:
Probably also worthwhile to comment somewhere that we're not actually doing any protocol version switching, and that it should be implemented if we support multiple protocols at some point....

Reply:
The async PR requires RESP3 so we do have to support protocol switching.

Reply:
I have updated this PR to only support RESP2. RESP3 will be added in the "async" PR.